luarules/gadgets/unit_workertime_boost.lua

```diff
@@ -5,16 +5,16 @@ function gadget:GetInfo()
 		author = "SethDGamre",
 		date = "April 2024",
 		license = "Public domain",
-		layer = 1,
+		layer = 50,
```

Comment:
why change and why specifically to 50 (especially since there's already a gadget at that layer)?

Reply:
unit_explosion_spawner is on that layer and so is unit_evolution gadget. I don't know how layers work, but I just read something written by HoboJoe about how it's good practice to start with a higher number and move it down if it needs prioritization. I don't know if it needs prioritization yet.

Reply:
changed to layer 0 as requested